doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -2715,8 +2360,7 @@ __ `Scalar variables`_
 Keywords can also return values so that they can be assigned into
 several `scalar variables`_ at once, into `a list variable`__, or
 into scalar variables and a list variable. All these usages require
-that returned values are Python lists or tuples or
-in Java arrays, Lists, or Iterators.
+that returned values are Python lists or tuples.
```

Comment:
Actually any list-like object is fine. Perhaps this should say something like "that returned values are lists or list-like objects".